mteb/tasks/Clustering/deu/BlurbsClusteringP2P.py

```diff
@@ -77,5 +84,28 @@ class BlurbsClusteringP2PFast(AbsTaskClustering):
     )
 
     def dataset_transform(self):
-        ds = clustering_downsample(self.dataset, self.seed)
-        self.dataset = ds
+        ds = dict()
+        for split in self.metadata.eval_splits:
+            labels = list(itertools.chain.from_iterable(self.dataset[split]["labels"]))
```

Comment:
`self.dataset[split]["labels"]` is already a list no?

Reply:
Ya. My goal is to use code from my previous "convert" PR which works, then refactor all instances in a separate PR.